packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type ConfirmWithLinkProps = {
+  confirmWithLinkPageStrings: ConfirmWithLinkPageStrings;
+  email: string;
+  // TODO : update type definition once callback function is defined
+  goBackCallback?: () => void;
+  isOpenWebmailButtonVisible?: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+  resendEmailCallback: () => void;
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+export type ConfirmWithLinkPageStrings = {
+  headingText: string;
+  headingFtlId: string;
+  instructionText: string;
+  instructionFtlId: string;
+};
+
+const ConfirmWithLink = ({
+  confirmWithLinkPageStrings,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible = false,
+  resendEmailCallback,
+}: ConfirmWithLinkProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedAriaLabel = ftlMsgResolver.getMsg(
+    'confirm-signup-aria-label',
+    'An enveloppe containing a link'
+  );
+
+  // TO-DO: Replace utility that gets these values by matching the email provider via regex.
+  const getWebmailValues = (email: string) => {
+    // TODO replace hardcoded email provider and link - create a utility for this, see open-webmail-mixin.js
+    const emailProvider = 'Gmail';
+    return {
+      emailProvider,
+      link: `https://mail.google.com/mail/u/?authuser=${email}`,
```

Comment:
Q: do we need to encode the email address?

Reply:
Might be something we can kick down the road to when we're doing functionality (if we add a TODO etc)

Reply:
Comment added.